pkg/k8sutil/kustomization.go

```diff
@@ -38,12 +38,20 @@ func (s kustPatches) Less(i, j int) bool {
 }
 
 func WriteKustomizationToFile(kustomization kustomizetypes.Kustomization, file string) error {
-	// we remove newTags from here... because...
 	cleanedImages := []kustomizetypes.Image{}
+
+	// Remove tags and deduplicate image list.
+	// Tags are removed because we don't want kustomize to change tags, only image names
```

Comment:
should this be true for digests as well?

Reply:
The comment refers to this line `image.NewTag = ""` (which isn't new), so I guess digests would be there too.

Reply:
Actually, no.  Kustomize image has a separate field for `Digest`.

Reply:
yeah, you shouldn't be able to change a digest anyways now that i think about it.